oauth2client/service_account.py

```diff
@@ -122,6 +122,82 @@ def create_scoped(self, scopes):
                                       **self._kwargs)
 
 
+class _ServiceAccountJwtHeaderCredentials(AssertionCredentials):
+  """Class representing a service account (signed JWT) credential."""
+
+  MAX_TOKEN_LIFETIME_SECS = 3600 # 1 hour in seconds
+
+  def __init__(self, service_account_id, service_account_email, private_key_id,
+               private_key_pkcs8_text, scopes, user_agent=None,
```

Comment:
For the JwtHeader credential, I would expect that scopes are not present as an argument.